Cargo.toml

```
@@ -18,25 +18,26 @@ exclude = [
 ]
 
 [dependencies]
-dusk-pki = {version = "0.6", default-features = false}
-dusk-poseidon = {version = "0.18", default-features = false }
-dusk-bls12_381 = {version = "0.6", default-features = false}
-dusk-jubjub = {version = "0.8", default-features = false}
-dusk-plonk = {version = "0.5", features = ["trace-print"], optional = true}
+dusk-pki = {version = "0.7.0-rc.0", default-features = false}
+dusk-poseidon = {version = "0.21.0-rc.0", default-features = false }
+dusk-bls12_381 = {version = "0.8.0-rc.0", default-features = false}
+dusk-jubjub = {version = "0.10.0-rc.0", default-features = false}
+dusk-plonk = {version = "0.8.0-rc.1", optional = true}
 num-bigint = {version = "0.3", optional = true } 
 num-traits = {version = "0.2", optional = true } 
-plonk_gadgets = {version = "0.5", optional = true}
-rand_core = {version = "0.5", default-features = false}  
-lazy_static = "1"
-rand = {version = "0.7", default-features = false, optional = true }
-canonical = { version = "0.5", optional = true }
-canonical_derive = { version = "0.5", optional = true }
+plonk_gadgets = {version = "0.6.0-rc.0", optional = true}
+rand_core = {version = "0.6", default-features = false}  
+lazy_static = "1.4"
+canonical = { version = "0.6", optional = true }
+canonical_derive = { version = "0.6", optional = true }
+rand = {version = "0.8"}
 anyhow = {version = "1", optional = true}
 dusk-bytes = "0.1"
 cfg-if = "1.0"
 
 [dev-dependencies]
-canonical_host = "0.5"
+microkelvin = "0.7.1"
```

Always use the form `major.minor` never use the `fix`:
```suggestion
microkelvin = "0.7"
```